e2e/actions/l2_challenger_test.go

```diff
@@ -99,13 +114,13 @@ func TestChallenger(gt *testing.T) {
 	validator.ActDeposit(t, 1_000)
 	includeL1Block(t, miner, validator.address)
 
-	require.Equal(t, proposer.SyncStatus().UnsafeL2, proposer.SyncStatus().FinalizedL2)
-
-	outputOracleContract, err := bindings.NewL2OutputOracle(sd.DeploymentsL1.L2OutputOracleProxy, miner.EthClient())
+	// check validator balance increased
+	bal, err := valPoolContract.BalanceOf(nil, validator.address)
 	require.NoError(t, err)
+	require.Equal(t, 0, big.NewInt(1_000).Cmp(bal))
```

Comment:
Could you make a var for `1_000`(e.g, depositAmount) and use it? This isn't a same value with `challengerInitialAmount` right?